pkg/trace/agent/normalizer.go

```diff
@@ -158,6 +158,14 @@ func (a *Agent) normalize(ts *info.TagStats, s *pb.Span) error {
 			delete(s.Meta, "http.status_code")
 		}
 	}
+
+	if len(s.SpanLinks) > 0 {
+		for _, link := range s.SpanLinks {
+			if val, ok := link.Attributes["link.name"]; ok {
+				link.Attributes["link.name"], _ = traceutil.NormalizeName(val)
```

Comment:
We probably shouldn't be silently ignoring this error, right? At the very least, we should consider logging the error with a bit of context about which span name is invalid? e.g.

```
var err error
link.Attributes["link.name"], err = traceutil.NormalizeName(val)
if err != nil {
    log.Debugf("Fixing malformed trace. 'link.name' attribute in span link invalid (reason=%q), setting link.Attributes["link.name"]=%s", err, link.Attributes["link.name"])
}
```

Reply:
Good catch. We should log something here